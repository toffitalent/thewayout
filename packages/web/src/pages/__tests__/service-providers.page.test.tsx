import { render } from '@test';
import ServiceProvidersPage from '../service-providers.page';

describe('ServiceProvidersPage', () => {
  test('renders correctly', () => {
    const { container } = render(<ServiceProvidersPage />);
    expect(container).toMatchSnapshot();
  });
});
