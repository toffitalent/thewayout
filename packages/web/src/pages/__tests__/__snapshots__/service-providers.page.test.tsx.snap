// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ServiceProvidersPage renders correctly 1`] = `
<div>
  <section
    class="hero"
  >
    <div
      aria-hidden="true"
      class="bg"
    >
      <span
        style="box-sizing: border-box; display: block; overflow: hidden; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px; position: absolute; top: 0px; left: 0px; bottom: 0px; right: 0px;"
      >
        <img
          alt=""
          data-nimg="fill"
          decoding="async"
          sizes="100vw"
          src="/_next/image?url=%2Ftest-file-stub&w=3840&q=75"
          srcset="/_next/image?url=%2Ftest-file-stub&w=640&q=75 640w, /_next/image?url=%2Ftest-file-stub&w=750&q=75 750w, /_next/image?url=%2Ftest-file-stub&w=828&q=75 828w, /_next/image?url=%2Ftest-file-stub&w=1080&q=75 1080w, /_next/image?url=%2Ftest-file-stub&w=1200&q=75 1200w, /_next/image?url=%2Ftest-file-stub&w=1920&q=75 1920w, /_next/image?url=%2Ftest-file-stub&w=2048&q=75 2048w, /_next/image?url=%2Ftest-file-stub&w=3840&q=75 3840w"
          style="position: absolute; top: 0px; left: 0px; bottom: 0px; right: 0px; box-sizing: border-box; padding: 0px; margin: auto; display: block; width: 0px; height: 0px; min-width: 100%; max-width: 100%; min-height: 100%; max-height: 100%; object-fit: cover;"
        />
      </span>
      s
    </div>
    <div
      class="container"
    >
      <div
        class="container justifyContent-center textAlign-center"
      >
        <div
          class="item bp-xs-12 bp-md-8"
        >
          <div
            class="alignItems-center flexDirection-row justifyContent-center marginBottom-5"
          >
            <p
              class="text marginRight-1 fontWeight-700"
            >
              Funded by
            </p>
            <span
              style="box-sizing: border-box; display: inline-block; overflow: hidden; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px; position: relative; max-width: 100%;"
            >
              <span
                style="box-sizing: border-box; display: block; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px; max-width: 100%;"
              >
                <img
                  alt=""
                  aria-hidden="true"
                  src="data:image/svg+xml,%3csvg%20xmlns=%27http://www.w3.org/2000/svg%27%20version=%271.1%27%20width=%2770%27%20height=%2724%27/%3e"
                  style="display: block; max-width: 100%; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px;"
                />
              </span>
              <img
                alt="google"
                data-nimg="intrinsic"
                decoding="async"
                src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
                style="position: absolute; top: 0px; left: 0px; bottom: 0px; right: 0px; box-sizing: border-box; padding: 0px; margin: auto; display: block; width: 0px; height: 0px; min-width: 100%; max-width: 100%; min-height: 100%; max-height: 100%;"
              />
              <noscript />
            </span>
          </div>
          <h1>
            Maximize Your Community Impact
          </h1>
          <p
            class="text"
          >
            Our algorithm matches job seekers with local service providers that can help with their support and professional needs, keeping families together.
          </p>
          <div
            class="actions"
          >
            <a
              class="button md solid c-primary"
              href="mailto:support@twout.org"
            >
              Add Your Organization
              <span
                aria-hidden="true"
                class="accessoryRight"
              >
                <svg
                  aria-hidden="true"
                  class="icon"
                  focusable="false"
                  stroke-width="2"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M0 0h24v24H0z"
                    stroke="none"
                  />
                  <path
                    d="M5 12h14M15 16l4-4M15 8l4 4"
                  />
                </svg>
              </span>
            </a>
          </div>
        </div>
      </div>
    </div>
  </section>
  <div
    class="section sectionServiceProviders paddingRight-5 paddingLeft-5"
  >
    <h2
      class="text sectionTitle1 marginBottom-12"
    >
      Our Service Providers
    </h2>
    <div
      class=""
    >
      <div
        class="container justifyContent-center"
      >
        <div
          class="serviceProviderImg item bp-xs-3 bp-lg-true justifyContent-center display-flex height-10 marginTop-3"
        >
          <span
            style="box-sizing: border-box; display: inline-block; overflow: hidden; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px; position: relative; max-width: 100%;"
          >
            <span
              style="box-sizing: border-box; display: block; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px; max-width: 100%;"
            >
              <img
                alt=""
                aria-hidden="true"
                src="data:image/svg+xml,%3csvg%20xmlns=%27http://www.w3.org/2000/svg%27%20version=%271.1%27%20width=%270%27%20height=%270%27/%3e"
                style="display: block; max-width: 100%; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px;"
              />
            </span>
            <img
              alt=""
              data-nimg="intrinsic"
              decoding="async"
              src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
              style="position: absolute; top: 0px; left: 0px; bottom: 0px; right: 0px; box-sizing: border-box; padding: 0px; margin: auto; display: block; width: 0px; height: 0px; min-width: 100%; max-width: 100%; min-height: 100%; max-height: 100%; object-fit: contain;"
            />
            <noscript />
          </span>
        </div>
        <div
          class="serviceProviderImg item bp-xs-3 bp-lg-true justifyContent-center display-flex height-10 marginTop-3"
        >
          <span
            style="box-sizing: border-box; display: inline-block; overflow: hidden; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px; position: relative; max-width: 100%;"
          >
            <span
              style="box-sizing: border-box; display: block; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px; max-width: 100%;"
            >
              <img
                alt=""
                aria-hidden="true"
                src="data:image/svg+xml,%3csvg%20xmlns=%27http://www.w3.org/2000/svg%27%20version=%271.1%27%20width=%270%27%20height=%270%27/%3e"
                style="display: block; max-width: 100%; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px;"
              />
            </span>
            <img
              alt=""
              data-nimg="intrinsic"
              decoding="async"
              src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
              style="position: absolute; top: 0px; left: 0px; bottom: 0px; right: 0px; box-sizing: border-box; padding: 0px; margin: auto; display: block; width: 0px; height: 0px; min-width: 100%; max-width: 100%; min-height: 100%; max-height: 100%; object-fit: contain;"
            />
            <noscript />
          </span>
        </div>
        <div
          class="serviceProviderImg item bp-xs-3 bp-lg-true justifyContent-center display-flex height-10 marginTop-3"
        >
          <span
            style="box-sizing: border-box; display: inline-block; overflow: hidden; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px; position: relative; max-width: 100%;"
          >
            <span
              style="box-sizing: border-box; display: block; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px; max-width: 100%;"
            >
              <img
                alt=""
                aria-hidden="true"
                src="data:image/svg+xml,%3csvg%20xmlns=%27http://www.w3.org/2000/svg%27%20version=%271.1%27%20width=%270%27%20height=%270%27/%3e"
                style="display: block; max-width: 100%; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px;"
              />
            </span>
            <img
              alt=""
              data-nimg="intrinsic"
              decoding="async"
              src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
              style="position: absolute; top: 0px; left: 0px; bottom: 0px; right: 0px; box-sizing: border-box; padding: 0px; margin: auto; display: block; width: 0px; height: 0px; min-width: 100%; max-width: 100%; min-height: 100%; max-height: 100%; object-fit: contain;"
            />
            <noscript />
          </span>
        </div>
        <div
          class="serviceProviderImg item bp-xs-3 bp-lg-true justifyContent-center display-flex height-10 marginTop-3"
        >
          <span
            style="box-sizing: border-box; display: inline-block; overflow: hidden; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px; position: relative; max-width: 100%;"
          >
            <span
              style="box-sizing: border-box; display: block; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px; max-width: 100%;"
            >
              <img
                alt=""
                aria-hidden="true"
                src="data:image/svg+xml,%3csvg%20xmlns=%27http://www.w3.org/2000/svg%27%20version=%271.1%27%20width=%270%27%20height=%270%27/%3e"
                style="display: block; max-width: 100%; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px;"
              />
            </span>
            <img
              alt=""
              data-nimg="intrinsic"
              decoding="async"
              src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
              style="position: absolute; top: 0px; left: 0px; bottom: 0px; right: 0px; box-sizing: border-box; padding: 0px; margin: auto; display: block; width: 0px; height: 0px; min-width: 100%; max-width: 100%; min-height: 100%; max-height: 100%; object-fit: contain;"
            />
            <noscript />
          </span>
        </div>
        <div
          class="serviceProviderImg item bp-xs-3 bp-lg-true justifyContent-center display-flex height-10 marginTop-3"
        >
          <span
            style="box-sizing: border-box; display: inline-block; overflow: hidden; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px; position: relative; max-width: 100%;"
          >
            <span
              style="box-sizing: border-box; display: block; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px; max-width: 100%;"
            >
              <img
                alt=""
                aria-hidden="true"
                src="data:image/svg+xml,%3csvg%20xmlns=%27http://www.w3.org/2000/svg%27%20version=%271.1%27%20width=%270%27%20height=%270%27/%3e"
                style="display: block; max-width: 100%; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px;"
              />
            </span>
            <img
              alt=""
              data-nimg="intrinsic"
              decoding="async"
              src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
              style="position: absolute; top: 0px; left: 0px; bottom: 0px; right: 0px; box-sizing: border-box; padding: 0px; margin: auto; display: block; width: 0px; height: 0px; min-width: 100%; max-width: 100%; min-height: 100%; max-height: 100%; object-fit: contain;"
            />
            <noscript />
          </span>
        </div>
        <div
          class="serviceProviderImg item bp-xs-3 bp-lg-true justifyContent-center display-flex height-10 marginTop-3"
        >
          <span
            style="box-sizing: border-box; display: inline-block; overflow: hidden; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px; position: relative; max-width: 100%;"
          >
            <span
              style="box-sizing: border-box; display: block; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px; max-width: 100%;"
            >
              <img
                alt=""
                aria-hidden="true"
                src="data:image/svg+xml,%3csvg%20xmlns=%27http://www.w3.org/2000/svg%27%20version=%271.1%27%20width=%270%27%20height=%270%27/%3e"
                style="display: block; max-width: 100%; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px;"
              />
            </span>
            <img
              alt=""
              data-nimg="intrinsic"
              decoding="async"
              src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
              style="position: absolute; top: 0px; left: 0px; bottom: 0px; right: 0px; box-sizing: border-box; padding: 0px; margin: auto; display: block; width: 0px; height: 0px; min-width: 100%; max-width: 100%; min-height: 100%; max-height: 100%; object-fit: contain;"
            />
            <noscript />
          </span>
        </div>
        <div
          class="serviceProviderImg item bp-xs-3 bp-lg-true justifyContent-center display-flex height-10 marginTop-3"
        >
          <span
            style="box-sizing: border-box; display: inline-block; overflow: hidden; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px; position: relative; max-width: 100%;"
          >
            <span
              style="box-sizing: border-box; display: block; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px; max-width: 100%;"
            >
              <img
                alt=""
                aria-hidden="true"
                src="data:image/svg+xml,%3csvg%20xmlns=%27http://www.w3.org/2000/svg%27%20version=%271.1%27%20width=%270%27%20height=%270%27/%3e"
                style="display: block; max-width: 100%; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px;"
              />
            </span>
            <img
              alt=""
              data-nimg="intrinsic"
              decoding="async"
              src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
              style="position: absolute; top: 0px; left: 0px; bottom: 0px; right: 0px; box-sizing: border-box; padding: 0px; margin: auto; display: block; width: 0px; height: 0px; min-width: 100%; max-width: 100%; min-height: 100%; max-height: 100%; object-fit: contain;"
            />
            <noscript />
          </span>
        </div>
      </div>
    </div>
  </div>
  <section
    class=""
  >
    <div
      class="container flexDirection-row-reverse"
    >
      <div
        class="sectionContent bp-xs-12 bp-md-6 paddingTop-24 paddingBottom-24"
      >
        <h2
          class="text sectionTitle2 marginBottom-10"
        >
          Why partner with us?
        </h2>
        <div
          class="flexDirection-row"
        >
          <div
            class="flexDirection-column display-flex width-1/2"
          >
            <span
              style="box-sizing: border-box; display: inline-block; overflow: hidden; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px; position: relative; max-width: 100%;"
            >
              <span
                style="box-sizing: border-box; display: block; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px; max-width: 100%;"
              >
                <img
                  alt=""
                  aria-hidden="true"
                  src="data:image/svg+xml,%3csvg%20xmlns=%27http://www.w3.org/2000/svg%27%20version=%271.1%27%20width=%27160%27%20height=%27160%27/%3e"
                  style="display: block; max-width: 100%; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px;"
                />
              </span>
              <img
                alt=""
                data-nimg="intrinsic"
                decoding="async"
                src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
                style="position: absolute; top: 0px; left: 0px; bottom: 0px; right: 0px; box-sizing: border-box; padding: 0px; margin: auto; display: block; width: 0px; height: 0px; min-width: 100%; max-width: 100%; min-height: 100%; max-height: 100%; object-fit: contain;"
              />
              <noscript />
            </span>
            <p
              class="text marginTop-6 paddingRight-2"
            >
              Retention rate. When employers, clients, and SSPs are engaged and actively using the platform.
            </p>
          </div>
          <div
            class="flexDirection-column display-flex width-1/2"
          >
            <span
              style="box-sizing: border-box; display: inline-block; overflow: hidden; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px; position: relative; max-width: 100%;"
            >
              <span
                style="box-sizing: border-box; display: block; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px; max-width: 100%;"
              >
                <img
                  alt=""
                  aria-hidden="true"
                  src="data:image/svg+xml,%3csvg%20xmlns=%27http://www.w3.org/2000/svg%27%20version=%271.1%27%20width=%27160%27%20height=%27160%27/%3e"
                  style="display: block; max-width: 100%; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px;"
                />
              </span>
              <img
                alt=""
                data-nimg="intrinsic"
                decoding="async"
                src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
                style="position: absolute; top: 0px; left: 0px; bottom: 0px; right: 0px; box-sizing: border-box; padding: 0px; margin: auto; display: block; width: 0px; height: 0px; min-width: 100%; max-width: 100%; min-height: 100%; max-height: 100%; object-fit: contain;"
              />
              <noscript />
            </span>
            <p
              class="text marginTop-6"
            >
              Supportive Services. The average returning citizen requires minimum of 2 additional supportive services along with employment.
            </p>
          </div>
        </div>
        <p
          class="text marginTop-12 fontWeight-700"
        >
          As a member of our support network, you are integral to the success of our job seekers. Our holistic approach to supporting our clients through their personal and professional journey begins with you.
        </p>
      </div>
      <div
        class="imgWrapper bp-xs-12 bp-md-6"
      >
        <span
          style="box-sizing: border-box; display: block; overflow: hidden; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px; position: relative;"
        >
          <span
            style="box-sizing: border-box; display: block; background: none; opacity: 1; border: 0px; margin: 0px; padding: 100% 0px 0px 0px;"
          />
          <img
            alt=""
            data-nimg="responsive"
            decoding="async"
            src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
            style="position: absolute; top: 0px; left: 0px; bottom: 0px; right: 0px; box-sizing: border-box; padding: 0px; margin: auto; display: block; width: 0px; height: 0px; min-width: 100%; max-width: 100%; min-height: 100%; max-height: 100%; object-fit: cover;"
          />
          <noscript />
        </span>
      </div>
    </div>
  </section>
  <section
    class="section paddingRight-4 paddingLeft-4 paddingTop-36"
  >
    <p
      class="text fontSize-4xl fontWeight-700 textAlign-center"
    >
      Why Partner With Us?
    </p>
    <p
      class="text color-grey-600 marginTop-2 fontSize-xl fontWeight-700"
    >
      Our software helps you improve case management outcomes and increase funding opportunities.
    </p>
  </section>
  <section
    class=""
  >
    <div
      class="container flexDirection-row-reverse"
    >
      <div
        class="sectionContent bp-xs-12 bp-md-6"
      >
        <h2
          class="text sectionTitle3 marginBottom-5"
        >
          Modernize Your Case Management
        </h2>
        <p
          class="text"
        >
          Our dashboard allows you to scale your impact by focusing on clients that need your support the most. Manage case notes and communicate with your clients all in one place. No more notebooks and manilla folders!
        </p>
        <a
          class="button md solid c-primary marginTop-8"
          href="mailto:support@twout.org"
        >
          Add Organization
          <span
            aria-hidden="true"
            class="accessoryRight"
          >
            <svg
              aria-hidden="true"
              class="icon"
              focusable="false"
              stroke-width="2"
              viewBox="0 0 24 24"
            >
              <path
                d="M0 0h24v24H0z"
                stroke="none"
              />
              <path
                d="M5 12h14M15 16l4-4M15 8l4 4"
              />
            </svg>
          </span>
        </a>
      </div>
      <div
        class="imgWrapper bp-xs-12 bp-md-6"
      >
        <span
          style="box-sizing: border-box; display: block; overflow: hidden; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px; position: relative;"
        >
          <span
            style="box-sizing: border-box; display: block; background: none; opacity: 1; border: 0px; margin: 0px; padding: 100% 0px 0px 0px;"
          />
          <img
            alt=""
            data-nimg="responsive"
            decoding="async"
            src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
            style="position: absolute; top: 0px; left: 0px; bottom: 0px; right: 0px; box-sizing: border-box; padding: 0px; margin: auto; display: block; width: 0px; height: 0px; min-width: 100%; max-width: 100%; min-height: 100%; max-height: 100%; object-fit: cover;"
          />
          <noscript />
        </span>
      </div>
    </div>
  </section>
  <section
    class=""
  >
    <div
      class="container"
    >
      <div
        class="sectionContent bp-xs-12 bp-md-6"
      >
        <h2
          class="text sectionTitle3 marginBottom-5"
        >
          Receive New Client Referrals
        </h2>
        <p
          class="text"
        >
          Based on services provided and location, our algorithm matches you with new clients in your area. Or, you can onboard existing clients.
        </p>
        <a
          class="button md solid c-primary marginTop-8"
          href="mailto:support@twout.org"
        >
          Get Started
          <span
            aria-hidden="true"
            class="accessoryRight"
          >
            <svg
              aria-hidden="true"
              class="icon"
              focusable="false"
              stroke-width="2"
              viewBox="0 0 24 24"
            >
              <path
                d="M0 0h24v24H0z"
                stroke="none"
              />
              <path
                d="M5 12h14M15 16l4-4M15 8l4 4"
              />
            </svg>
          </span>
        </a>
      </div>
      <div
        class="imgWrapper bp-xs-12 bp-md-6"
      >
        <span
          style="box-sizing: border-box; display: block; overflow: hidden; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px; position: relative;"
        >
          <span
            style="box-sizing: border-box; display: block; background: none; opacity: 1; border: 0px; margin: 0px; padding: 100% 0px 0px 0px;"
          />
          <img
            alt=""
            data-nimg="responsive"
            decoding="async"
            src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
            style="position: absolute; top: 0px; left: 0px; bottom: 0px; right: 0px; box-sizing: border-box; padding: 0px; margin: auto; display: block; width: 0px; height: 0px; min-width: 100%; max-width: 100%; min-height: 100%; max-height: 100%; object-fit: cover;"
          />
          <noscript />
        </span>
      </div>
    </div>
  </section>
  <section
    class=""
  >
    <div
      class="container flexDirection-row-reverse"
    >
      <div
        class="sectionContent bp-xs-12 bp-md-6"
      >
        <h2
          class="text sectionTitle3 marginBottom-5"
        >
          Track Your Impact
        </h2>
        <p
          class="text"
        >
          Get access to real time analytics, create impact reports to share with funders, and track your client and organization's engagement metrics
        </p>
        <a
          class="button md solid c-primary marginTop-8"
          href="mailto:support@twout.org"
        >
          Add Organization
          <span
            aria-hidden="true"
            class="accessoryRight"
          >
            <svg
              aria-hidden="true"
              class="icon"
              focusable="false"
              stroke-width="2"
              viewBox="0 0 24 24"
            >
              <path
                d="M0 0h24v24H0z"
                stroke="none"
              />
              <path
                d="M5 12h14M15 16l4-4M15 8l4 4"
              />
            </svg>
          </span>
        </a>
      </div>
      <div
        class="imgWrapper bp-xs-12 bp-md-6"
      >
        <span
          style="box-sizing: border-box; display: block; overflow: hidden; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px; position: relative;"
        >
          <span
            style="box-sizing: border-box; display: block; background: none; opacity: 1; border: 0px; margin: 0px; padding: 100% 0px 0px 0px;"
          />
          <img
            alt=""
            data-nimg="responsive"
            decoding="async"
            src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
            style="position: absolute; top: 0px; left: 0px; bottom: 0px; right: 0px; box-sizing: border-box; padding: 0px; margin: auto; display: block; width: 0px; height: 0px; min-width: 100%; max-width: 100%; min-height: 100%; max-height: 100%; object-fit: cover;"
          />
          <noscript />
        </span>
      </div>
    </div>
  </section>
  <section
    class="section paddingRight-4 paddingLeft-4 paddingTop-36"
  >
    <div
      class="bp-xs-12 bp-md-6 alignItems-center flexDirection-column display-flex"
    >
      <p
        class="text color-primary marginBottom-2 fontSize-xl fontWeight-700"
      >
        Our Belief
      </p>
      <p
        class="text fontSize-3xl fontWeight-700 textAlign-center"
      >
        "It takes a village to raise a child, but it takes a community to support and uplift someone through the challenges and obstacles of life."
      </p>
    </div>
  </section>
  <section
    class=""
  >
    <div
      class="container flexDirection-row-reverse"
    >
      <div
        class="sectionContentWider bp-xs-12 bp-md-6"
      >
        <h2
          class="text title marginBottom-2"
        >
          How it works?
        </h2>
        <div
          class=""
        >
          <div
            class="infoItem borderBottom flexDirection-row paddingTop-10 paddingBottom-10"
          >
            <p
              class="text color-primary fontSize-5xl fontWeight-700 lineHeight-none"
            >
              1
            </p>
            <div
              class="marginLeft-8"
            >
              <p
                class="text marginBottom-2 fontSize-xl fontWeight-700"
              >
                Create Your Profile
              </p>
              <p
                class="text color-grey-600"
              >
                Add your organization's details; name, location, services provided, and contact info.
              </p>
            </div>
          </div>
          <div
            class="infoItem borderBottom flexDirection-row paddingTop-10 paddingBottom-10"
          >
            <p
              class="text color-primary fontSize-5xl fontWeight-700 lineHeight-none"
            >
              2
            </p>
            <div
              class="marginLeft-8"
            >
              <p
                class="text marginBottom-2 fontSize-xl fontWeight-700"
              >
                Receive Client Referrals
              </p>
              <p
                class="text color-grey-600"
              >
                Our algorithm matches you with clients seeking supportive services in your area.
              </p>
            </div>
          </div>
          <div
            class="infoItem borderBottom flexDirection-row paddingTop-10 paddingBottom-10"
          >
            <p
              class="text color-primary fontSize-5xl fontWeight-700 lineHeight-none"
            >
              3
            </p>
            <div
              class="marginLeft-8"
            >
              <p
                class="text marginBottom-2 fontSize-xl fontWeight-700"
              >
                Manage Client Journey
              </p>
              <p
                class="text color-grey-600"
              >
                Utilize our dashboard to help manage your client's personal and professional journey.
              </p>
            </div>
          </div>
        </div>
        <a
          class="button md solid c-primary"
          href="mailto:support@twout.org"
        >
          Register Organization
          <span
            aria-hidden="true"
            class="accessoryRight"
          >
            <svg
              aria-hidden="true"
              class="icon"
              focusable="false"
              stroke-width="2"
              viewBox="0 0 24 24"
            >
              <path
                d="M0 0h24v24H0z"
                stroke="none"
              />
              <path
                d="M5 12h14M15 16l4-4M15 8l4 4"
              />
            </svg>
          </span>
        </a>
      </div>
      <div
        class="imgWrapper bp-xs-12 bp-md-6"
      >
        <span
          style="box-sizing: border-box; display: block; overflow: hidden; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px; position: relative;"
        >
          <span
            style="box-sizing: border-box; display: block; background: none; opacity: 1; border: 0px; margin: 0px; padding: 100% 0px 0px 0px;"
          />
          <img
            alt=""
            data-nimg="responsive"
            decoding="async"
            src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
            style="position: absolute; top: 0px; left: 0px; bottom: 0px; right: 0px; box-sizing: border-box; padding: 0px; margin: auto; display: block; width: 0px; height: 0px; min-width: 100%; max-width: 100%; min-height: 100%; max-height: 100%; object-fit: cover;"
          />
          <noscript />
        </span>
      </div>
    </div>
  </section>
  <section
    class="greenBg section paddingRight-4 paddingLeft-4 paddingTop-36 paddingBottom-36"
  >
    <p
      class="text sectionTitle2 marginBottom-10 fontWeight-700 textAlign-center"
    >
      Explore our service provider resources.
    </p>
    <a
      class="button md solid c-primary"
      href="https://www.findhelp.org/"
      rel="noopener noreferrer"
      target="_blank"
    >
      Search For Additional Services
      <span
        aria-hidden="true"
        class="accessoryRight"
      >
        <svg
          aria-hidden="true"
          class="icon"
          focusable="false"
          stroke-width="2"
          viewBox="0 0 24 24"
        >
          <path
            d="M0 0h24v24H0z"
            stroke="none"
          />
          <path
            d="M5 12h14M15 16l4-4M15 8l4 4"
          />
        </svg>
      </span>
    </a>
  </section>
</div>
`;
